src/components/ShipSheet.vue

```diff
@@ -83,6 +71,14 @@ export default {
         console.log(error);
       });
     },
+  },
+  computed: {
+    modelString() {
+      return `I'm a <em>${this.model}</em> model`;
```

Comment:
What do you think about this @Dygerydoo?

I couldn't figure out some other better way to access the data value. When placing the raw html in the data function with the _double moustache_, it was rendered as it is; while JS interpolation returned _undefined_